### PR TITLE
Pic.topic.format controller

### PR DESCRIPTION
**Pull request by dreyerz**
Adds Pic, Topic, and Format APIs
Adds Seed data and pictures in wwwroot file

### CODE REVIEW

File: EventLite/Data/CatalogSeed.cs [hunk 1]
```diff
@@ -53,33 +51,123 @@ private static IEnumerable<CatalogEvent> GetPreconfiguredCatalogEvents()
...
-
                     VenueName = "Washington State Convention Center",
                     VenueAddressLine1 = "705 Pike St.",
                     VenueCity = "Seattle",
                     VenueStateProvince = "WA",
                     VenuePostalCode = "98101",
                     VenueMapUrl = "http://TODOREPLACEME/",
-
-
-                    // TODO: Understand why this syntax works for Venue property vs.
```
Comment (bpm150) [inline EventLite/Data/CatalogSeed.cs:65]: Please retain this comment, it is reminder for me to investigate

File: EventLite/Data/CatalogContext.cs [hunk 2]
```diff
@@ -32,9 +32,6 @@ public class CatalogContext : DbContext
         // How
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            // QUESTION:
```
Comment (bpm150) [inline EventLite/Data/CatalogContext.cs:35]: Please keep this comment, I will investigate

Comment (dreyerz) [reply EventLite/Data/CatalogContext.cs:35]: Added back

File: EventLite/Data/CatalogSeed.cs (hunk 1 above)
Comment (dreyerz) [reply EventLite/Data/CatalogSeed.cs:65]: Added comments back in

**Pull request merged**